src/femtomes.cpp

```diff
@@ -605,11 +617,32 @@ int GPSDriverFemto::configure(unsigned &baudrate, const GPSConfig &config)
 	}
 
 	if (_output_mode == OutputMode::GPS) {
+		if (writeAckedCommandFemto("LOG UAVGPSB 0.1\r\n", "<LOG OK", FEMTO_RESPONSE_TIMEOUT) == 0) {
+			FEMTO_DEBUG("Femto: command LOG UAVGPSB 0.1 success");
+
+		} else {
+			FEMTO_ERR("Femto: command LOG UAVGPSB 0.1 failed");
+		}
+
+		if (writeAckedCommandFemto("LOG UAVSTATUSB 0.1\r\n", "<LOG OK", FEMTO_RESPONSE_TIMEOUT) == 0) {
```

Comment:
The status could be enabled at a lower rate, to reduce link bandwidth. Probably 1 or 2 Hz is enough.
Also you could only enable it if `_satellite_info` is not null.